src/index.html

```diff
@@ -0,0 +1,241 @@
+<!DOCTYPE html>
+<html lang="en" class="no-js">
+
+<head>
+	<!-- Meta, title, CSS, favicons, etc. -->
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<meta name="description" content="">
+	<meta name="author" content="Micromata GmbH">
+	<title>Baumeister</title>
+
+	<!-- Bundled vendor CSS files -->
+	@@vendor.css
+
+	<!-- Our compiled and merged Sass files -->
+	@@app.css
+
+	<!--
+	Replace "no-js" class with "js" in case JavaScript is available.
+	You can delete this when you feel the need to make use of modernizr.
+	-->
+	<script>(function(h){h.className = h.className.replace('no-js', 'js')})(document.documentElement)</script>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.3/js/bootstrap.min.js"></script>
+  <link href="https://use.fontawesome.com/releases/v5.0.4/css/all.css" rel="stylesheet">
```

Comment:
Bitte nichts von einem CDN zu holen. 
Diese `link` und `script` Elemente können weg.

Bootstrap und jQuery liefert Baumeister von Hause aus.

fontawesome musst du via npm als dependency hinzufügen und dass CSS in unser Bundle aufnehmen
Siehe:
- https://github.com/micromata/baumeister#using-external-libraries
- https://github.com/micromata/baumeister#bundling-css-from-dependencies